Cucumber/README.md

```diff
@@ -0,0 +1,61 @@
+## Java-Cucumber-Junit
+
+### Environment Setup
+
+1. Global Dependencies
+    * Install [Maven](https://maven.apache.org/install.html)
+    * Or Install Maven with [Homebrew](http://brew.sh/) (Easier)
+    ```
+    $ brew install maven
+    ```
+2. Sauce Labs Credentials
+    * In the terminal, export your Sauce Labs credentials as environmental variables:
+    ```
+    $ export SAUCE_USERNAME=<your Sauce Labs username>
+    $ export SAUCE_ACCESS_KEY=<your Sauce Labs access key>
+    ```
+3. Project Dependencies
+    * Check that packages are available
+    ```
+    $ cd Java-CucumberJVM-JUnit
+    $ mvn test-compile
+    ```
+    * You may also want to run the command below to check for outdated dependencies. Please be sure to verify and review updates before editing your pom.xml file as they may not be compatible with your code.
+    ```
+    $ mvn versions:display-dependency-updates
+    ```
+    
+### Running the tests
+```
+$ make run_all_in_parallel
```

Comment:
in the other examples you just have `make test` maybe do the same here to keep it consistent.